src/block_cache/block_cache.c

```diff
@@ -871,6 +871,13 @@ __wt_blkcache_setup(WT_SESSION_IMPL *session, const char *cfg[], bool reconfig)
     if (cval.val == 0)
         return (0);
 
+    /* FIXME-WT-15663 Disable block cache before it's stable */
+    __wt_verbose_warning(session, WT_VERB_BLKCACHE,
+      "Block cache is currently disabled (wait WT-15663). Continuing without block cache, "
```

Comment:
We don't need to print tickets in the logs. And do we need to print `cval.val`?
```suggestion
      "Block cache is currently disabled. Continuing without block cache"
```